src/pages/Hwajong/Login/Login.js

```diff
@@ -1,11 +1,77 @@
-import React from 'react'
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './Login.scss';
 
-const Login = () => {
-    return (
-        <div>
-            
+function Login() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleIdInput = event => {
+    setEmail(event.target.value);
+  };
+
+  const handlePasswordInput = event => {
+    setPassword(event.target.value);
+  };
+
+  const [isActive, setIsActive] = useState(false);
+  const isSucceedLogin = () => {
+    return email.includes('@') && password.length > 4
+      ? setIsActive(true)
+      : setIsActive(false);
+  };
+
+  const navigate = useNavigate();
+
+  const goToMain = () => {
+    if (isActive === true) {
+      navigate('/hwajong-main');
+    }
+  };
+
+  return (
+    <div className="wrap">
+      <div className="container">
+        <h1>Westagram</h1>
+        <form>
+          <div className="inputWrapper">
+            <input
+              className="loginFormInput"
+              type="text"
+              id="id"
+              name="id"
+              onChange={handleIdInput}
+              onKeyUp={isSucceedLogin}
+              placeholder="전화번호, 사용자 이름 또는 이메일"
+            />
+          </div>
+          <div className="inputWrapper">
+            <input
+              className="passwordFormInput login-password"
+              type="password"
+              id="password"
+              placeholder="비밀번호"
+              onChange={handlePasswordInput}
+              onKeyUp={isSucceedLogin}
+            />
+          </div>
+          <div className="button">
+            <button
+              type="button"
+              className={isActive ? 'activeBtn' : 'unactiveBtn'}
+              disabled={email === '' || password === '' ? true : false}
```

Comment:
isActive로 사용 가능.
(state로 관리할 필요 X, email과 password state를 이용해 단순 변수로 선언해 사용)
```suggestion
              disabled={!isActive}
```